bootstrap-salt.sh

```diff
@@ -4461,7 +4461,11 @@ install_amazon_linux_ami_deps() {
     # Shim to figure out if we're using old (rhel) or new (aws) rpms.
     _USEAWS=$BS_FALSE
 
-    repo_rev="$(echo "${GIT_REV:-$STABLE_REV}"  | sed 's|.*\/||g')"
+    if [ "$ITYPE" = "stable" ]; then
+        repo_rev="$(echo "${STABLE_REV}"  | sed 's|.*\/||g')"
+    else
+        repo_rev="latest"
+    fi
```

Comment:
Yeah, that's fine. I just wanted to highlight that it seems like `$repo_rev` variable was used to guess the right path for the repo URL. This actually should honor exact version pinning for stable installations, e.g. `2016.11.4` and the feature doesn't work for Amazon Linux for now. But that's another issue :-)